content/sdk/10.js/00.ethers/20.v6/01.features.md

```diff
@@ -1,59 +1,50 @@
 ---
-title: ZKsync Era Features
-description:
+title: "zksync Era Features"
+description: "Explore zksync Era Features and Custom Transactions"
+tags: ["zksync", "ethereum", "zksync Era", "transactions", "paymasters"]
 ---
 
-While ZKsync is mostly `ethers`-compatible, it has some differences compared to Ethereum.
-The major of those are:
+zksync Era offers several unique features compared to Ethereum, which require some specific handling. This guide will
+help you understand these differences and how to work with them using the `zksync-ethers` library.
 
-- Account abstraction support (accounts may have near-arbitrary validation logic,
-  and also paymaster support is enabled).
-- Deployment transactions require the contracts' bytecode to be passed in a separate field.
-- The fee system is somewhat different.
+## Key Differences
 
-These require us to extend standard Ethereum transactions with new custom fields.
-Such extended transactions are called EIP712 transactions since [EIP712](https://eips.ethereum.org/EIPS/eip-712)
-is used to sign them. You can look at the internal structure of the EIP712 transactions
-[here](https://docs.zksync.io/zk-stack/concepts/transaction-lifecycle.html#eip-712-0x71).
+::card-group
+::card{title="Account Abstraction" icon="i-heroicons-puzzle-piece"}
+zksync supports account abstraction, allowing accounts to have custom validation logic and enabling paymaster support.
+::
+::card{title="Deployment Transactions" icon="i-heroicons-paper-airplane"}
+Contracts on zksync require bytecode to be passed in a separate field during deployment.
+::
+::card{title="Fee System" icon="i-heroicons-banknotes"}
+The fee system in zksync differs from Ethereum's, necessitating additional fields in transactions.
+::
+::
 
-This document will focus solely on how to pass these arguments to the SDK.
+## EIP712 Transactions
 
-## Overrides
+To accommodate these differences, zksync extends standard Ethereum transactions with new fields, creating what are known
+as EIP712 transactions. More details on EIP712 transactions can be found [here](https://docs.zksync.io/zk-stack/concepts/transaction-lifecycle.html#eip-712-0x71).
 
-`ethers.js` has a notion of overrides. For any on-chain transaction, `ethers.js` finds the
-optimal `gasPrice`, `gasLimit`, `nonce`, and other important fields under the hood.
-But sometimes, you may have a need to explicitly provide these values
-(you want to set a smaller `gasPrice` for instance, or sign a transaction with future `nonce`).
+## Overrides in Transactions
 
-In this case, you can provide an `Overrides` object as the last parameter.
-There you can supply fields like `gasPrice`, `gasLimit`, `nonce` etc.
+`ethers.js` provides overrides for transactions, allowing you to set values like `gasPrice`, `gasLimit`, and `nonce`.
+zksync-specific fields are added using the `customData` object in these overrides.
```

Comment:
This feels misdirecting with the wording. Yes it is added using `customData` to set these properties, but you removed the example and description that the `customData` has to be a property of the `overrides` key.